sale_import_base/models/sale_order.py

```diff
@@ -0,0 +1,13 @@
+#  Copyright (c) Akretion 2020
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    si_amount_untaxed = fields.Float("(technical) Untaxed amount from import")
+    si_amount_tax = fields.Float("(technical) Tax amount from import")
+    si_amount_total = fields.Float("(technical) Total amount from import")
+    si_force_invoice_date = fields.Date("(technical) Forced invoice date")
+    si_force_invoice_number = fields.Char("(technical) Forced invoice number")
```

Comment:
@florian-dacosta Does these "si_" prefixes come from ShopInvider? If it's the case, it could be better to rename them "imported_" or something explicit with no link with Shopinvider.

Reply:
I've never used the module in production, so you're more up to decide what's the best with these data.

Intuitively it looks like it can be useful to compare the imported amounts with the computed amounts during the SO creation with Odoo taxes (in case of cents differences for example).

But if it's not that useful to have these info on the SO, I imagine the user can always go back to the payload raw data in case he needs to make these comparatives.